draft-ietf-shmoo-remote-fee.md

```diff
@@ -91,19 +90,18 @@ This principle aims to support the openness principle of the IETF as defined in
    documents, our WG mailing lists, our attendance lists, and our
    meeting minutes publicly available on the Internet."
    
-It should be noted that openness as defined in {{!RFC3935}} should be seen as open and free. 
 While the principle in RFC3935 is explicitly noting that this principle includes a requirement to open 
 basically all our documents and documentation and making them accessible over the Internet, it was
 probably written with mainly having email interactions in mind when talking about participation.
 This document extends this principle to explicitly cover online
-participation at meetings.
+participation at meetings. Particularly in this context, openness should be seen as open and free. 
 
-This document does not advise that all IETF meetings or events must have a remote participation option,
+This document does not stipulate that all IETF meetings or related IETF events must have a remote participation option,
 because there could be technical or other reasons why that might not always be possible.
 This document rather says that if remote participation is provided,
 there should always be a free option to make the process as open as possible. Having said that,
-it is of course strongly anticipated that at least all sessions of the main agenda of an IETF
-plenary meeting provide an option for remote participation.
+it is of course strongly anticipated that at least all working group sessions as well as BoFs
```

Comment:
The Hackathon is part of the regular IETF meeting and is included in the main agenda of the IETF meeting. It was therefore covered by the previous text but not by this proposed change. I suggest adding it to the list if this change is made.

Reply:
Perhaps
```
it is of course strongly anticipated that
all items on the main IETF agenda (working group sessions, BoFs, hackathon, etc)..
```

?

Reply:
Yes, that works for me.